Automattic/jetpack
# Support links: link directly to WordPress.com for WoA sites

Fixes 148-gh-atomic

#### Changes proposed in this Pull Request:

* On WoA sites, we can offer live chat support instead of asking folks to go through debug steps on their own.

**Note**

In the future, maybe we should consider linking self-hosted sites to Tools > Site Health instead?

#### Jetpack product discussion

* See 148-gh-atomic

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* On a self-hosted site, go to the Plugins menu. You should be redirected to the debugger.
* Repeat on a WoA site; you should be redirected to WordPress.com for a live chat session.
